### PR TITLE
removed month and year from units

### LINKED ISSUE #177: Remove months and years from default `units`
Currently, the `units` option defaults to including months and years.

This line:

https://github.com/EvanHahn/HumanizeDuration.js/blob/6029e9807718a7da2cbb1f859b3e7e0f94049069/humanize-duration.js#L1003

Should be changed to `["w", "d", "h", "m", "s"]`.

This pull request should be made against the `v4.0.0` branch, which you can see in #176.

### PR DESCRIPTION
**Pull request by morrme**
resolves #177

### CODE REVIEW

**Comment by morrme** [comment]
updating the tests to remove the units as well.

**Comment by morrme** [comment]
@EvanHahn there might be a typo in the current README at line 156. It has the expected outcome commented out in a nonstandard place. 

I didn't want to fix it if it was intentional , but that example should change as well. With the change requested by #177, the "unitMeasures" example changes to "57 weeks, 1 day". 

If desired, I can change the expected output to this value, or add the "units" option to the example so that it reflects the current expected output.

File: README.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ humanizeDuration(260040000, { spacer: "" }); // '3days, 14minutes'
 Number representing the maximum number of units to display for the duration.
 
 ```js
-humanizeDuration(1000000000000); // '31 years, 8 months, 1 week, 19 hours, 46 minutes, 40 seconds'
-humanizeDuration(1000000000000, { largest: 2 }); // '31 years, 8 month'
+humanizeDuration(1000000000000); // '1653 weeks, 3 days, 1 hour, 46 minutes, 40 seconds'
```
Comment (EvanHahn) [inline README.md:81]: Could you change this to a smaller value? Maybe `10000000000` (which should humanize to "16 weeks, 3 days, 17 hours, 46 minutes, 40 seconds")?

File: test/humanizer.js [hunk 2]
```diff
@@ -130,11 +130,11 @@ describe("humanizer", function () {
...
     const h = humanizer({ round: true });
 
-    assert.strictEqual(h(3692131200000, { largest: 1 }), "117 years");
-    assert.strictEqual(h(3692131200000, { largest: 2 }), "117 years");
+    assert.strictEqual(h(3692131200000, { largest: 1 }), "6105 weeks");
+    assert.strictEqual(h(3692131200000, { largest: 2 }), "6104 weeks, 5 days");
     assert.strictEqual(
       h(3692131200001, { largest: 100 }),
-      "116 years, 11 months, 4 weeks, 1 day, 4 hours, 30 minutes"
+      "6104 weeks, 5 days"
```
Comment (EvanHahn) [inline test/humanizer.js:137]: I'm surprised by this value...I would expect weeks, days, hours, minutes, etc. Is something broken with `largest`?

**Comment by EvanHahn** [comment]
You're right—that comment _is_ in a weird place. Would you mind opening a separate pull request for that? (If not, I can do it.)

I think it makes sense to update the expected output, not the `units` option.

File: test/humanizer.js (hunk 2 above)
Comment (morrme) [reply test/humanizer.js:137]: Seems to be due to the {round: true} in the constructor (`const h = humanizer({ round: true })`)
Without rounding  I get "6104 weeks, 5 days, 0.001 seconds"
With rounding it goes to "6104 weeks, 5 days"

**Comment by morrme** [comment]
@EvanHahn would you like the fix for the comment to be submitted in a PR against the current branch, and the fix with the updated output to be included in this one for v4.0.0?

**Comment by EvanHahn** [comment]
@morrme Yeah, that sounds perfect. I'll take another look when you've pushed changes.

File: README.md [hunk 3]
```diff
@@ -141,14 +141,14 @@ Customize the value used to calculate each unit of time.
 ```js
 humanizeDuration(400); // '0.4 seconds'
 humanizeDuration(400, {
-  // '1 year, 1 month, 5 days'
+  '
```
Comment (EvanHahn) [inline README.md:144]: This looks wrong—could we remove it?

**Pull request merged**